src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -928,6 +929,7 @@ App::DocumentObjectExecReturn *Spiral::execute(void)
 
         gp_Pnt2d aPnt(0, 0);
         gp_Dir2d aDir(2. * M_PI, myPitch);
+        Standard_Real coneDir = 1.0;

```

Comment:
It's unclear to me why this has been broken out as it's own variable. As it is set to `1.0`, and the only places it is used it is a multiplication factor, it seems better to just get rid of it altogether.